Source/Particles/Resampling/VelocityCoincidenceThinning.H

```diff
@@ -200,5 +218,6 @@ private:
     int m_ntheta, m_nphi;
     amrex::ParticleReal m_delta_ur;
     amrex::Vector<amrex::ParticleReal> m_delta_u;
+    amrex::ParticleReal m_max_w = 0;
```

Comment:
Could this default to `std::numeric_limits<amrex::ParticleReal>::max()`? This would simplify the checks of the weight.